server/trillian_log_server/main.go

```diff
@@ -68,6 +70,9 @@ func main() {
 	glog.CopyStandardLogTo("WARNING")
 	glog.Info("**** Log RPC Server Starting ****")
 
+	// Enable dumping of metrics to the log at regular interval.
+	go metric.DumpToLog(context.Background(), 10*time.Second)
```

Comment:
Can we have an expvar export as well? Followup is fine. And a flag to control log frequency would be handy for my testing.